included/README.md

```diff
@@ -142,13 +142,13 @@ $ docker run -it -v $PWD:/e2e -w /e2e -e DEBUG=cypress:* cypress/included:9.4.1
 These images have its entry point set to `cypress run` without any additional arguments. You can specify additional Cypress CLI arguments after the image name. For example to print the Help menu for the `cypress run` command:
 
 ```shell
-$ docker run -it --entrypoint=cypress cypress/included:9.4.1 --help
+$ docker run -it --entrypoint=cypress cypress/included:9.4.1 run --help
```

Comment:
One is for `docker` the other one for `cypress` so ...
`docker run container-image-for-cypress`

Once container is running, "inside it" we execute ...
`cypress run`

Otherwise/ right now example just executes ...
`cypress` 
Which will not get the help for `cypress run`